src/Views/WhatsNewPage.xaml

```diff
@@ -81,10 +82,122 @@
                     </StackPanel>
                 </Grid>
 
+
                 <muxc:ItemsRepeater
-                    Name="FeaturesContainer"
+                    Name="BigFeaturesContainer"
                     Grid.Row="1"
-                    Margin="35, 45, 45, 45"
+                    Margin="35, 5, 45, 10"
+                    animations:Connected.ListItemElementName="itemThumbnail"
+                    animations:Connected.ListItemKey="animationKeyContentGrid"
+                    ItemsSource="{x:Bind ViewModel.BigSource,Mode=OneWay}">
+
+                    <muxc:ItemsRepeater.Resources>
+                        <models:WhatsNewCard 
+                            x:Uid="WhatsNewPage_ExtensionsCard"
+                            x:Key="WhatsNewPage_ExtensionsCard"
+                            DarkThemeImage="ms-appx:///Assets/WhatsNewPage/DarkTheme/DashboardHero.png"
+                            LightThemeImage="ms-appx:///Assets/WhatsNewPage/LightTheme/DashboardHero.png"
+                            PageKey="DevHome.Dashboard.ViewModels.ExtensionsViewModel"

```

Comment:
Make this DevHome.ExtensionLibrary.ViewModels.ExtensionLibraryViewModel and the link should work. (You can rebase onto a newer version of the feature branch to double check)